code.js

```diff
@@ -1,3 +1,21 @@
 are_isomorphic(graph1, graph2) {
-    return undefined;
+    if (graph1.vertices.length !== graph2.vertices.length || graph1.edges.length !== graph2.edges.length) {
+    return false;
+}
+
+for (let vertex of graph1.vertices) {
+    let degree1 = graph1.degree(vertex);
+    let found = false;
+    for (let vertex2 of graph2.vertices) {
+        if (graph2.degree(vertex2) === degree1) {
```

Comment:
You can't just rely on the degree of the nodes, you have to check the actual connectivity.